### ISSUE TITLE
Some products are not appearing in search

### ISSUE THREAD

**Issue by sebbacon** [ebmdatalab/openprescribing#340]
Some presentations of Fluticasone Furoate/Vilanterol are not appearing as products in our search, despite having values in our data:

```
> select presentation_code, count(*) from frontend_prescription where presentation_code like '0301011Y0%' group by presentation_code;

┌───────────────────┬───────┐
│ presentation_code │ count │
├───────────────────┼───────┤
│ 0301011Y0AAAAAA   │  1466 │
│ 0301011Y0AAABAB   │  4505 │
│ 0301011Y0BBAAAA   │  1975 │
│ 0301011Y0BBABAB   │  5845 │
└───────────────────┴───────┘
(4 rows)

```

```
> select * from frontend_product where bnf_code like '0301011Y0%';

┌──────────┬──────┬────────────┐
│ bnf_code │ name │ is_generic │
├──────────┼──────┼────────────┤
└──────────┴──────┴────────────┘

```

This issue is related to #214

**Comment by sebbacon** [comment]
The reason:

* Our Product list is created (and appended to) each month from the NHS BNF code list
* This has items added and removed at least quarterly
* Relvar Ellipta was miscoded by the NHSBSA during 2014; codes changed in Jan 15, and again in Nov 16
* We only started ingesting the BNF data in 2015, so have missed out on being able to extract Products from before then

The only realistic fix is to source a master BNF code file that covers all of time since the data started.

If this isn't possible, then we can't make the data autocomplete by product for recoded items like this.

**Comment by richiecroker** [comment]
I will discuss with our BI team to find out if we have master downloads of BNF codes from previous months

**Comment by sebbacon** [comment]
@richiecroker  has access to monthly contemporaneous downloads going back to April 2013 (sample in email with subject **monthly contemporaneous downloads going back to April 2013**)

**Comment by sebbacon** [comment]
We've agreed to normalise the historic data to current BNF codes, but not try to fix new data heuristically.

Therefore the options for new data are:

* Continue to get data downloaded from ePACT from Rich's detail
* Get NHSBSA to send us the monthly data somehow
* Not update it on a continuing basis, just the historic data

**Comment by sebbacon** [comment]
This was closed by #380 as there are no longer products with the "old" codes in our dataset -- they are now all normalised to their current incarnations
